File: Beginner-c/1010.c
#include <stdio.h>

int main(){

    int pcode;
    double punits, pprice, valuetpay, total = 0;

    for(int i = 0; i < 2; i++){
        scanf("%d %lf %lf", &pcode, &punits, &pprice);
        valuetpay = (punits * pprice);
        total += valuetpay;
    }
    
    printf("VALOR A PAGAR: R$ %.2lf\n", total);
    return 0;
}

File: Beginner-Python/1010.py
total = 0

for _ in range(2):
    pcode = int(input())
    punits = float(input())
    pprice = float(input())
    valuetpay = (punits * pprice)
    total += valuetpay

print(f"VALOR A PAGAR: R$ {total:.2}\n")